sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_utils.py

```diff
@@ -0,0 +1,151 @@
+# The MIT License (MIT)
+# Copyright (c) 2014 Microsoft Corporation
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Internal Helper functions in the Azure Cosmos database change_feed service.
+"""
+
+import warnings
+from datetime import datetime
+from typing import Any, Dict, Tuple
+
+from azure.cosmos._change_feed.change_feed_state import ChangeFeedMode
+
+def add_args_to_kwargs(
+        args: Tuple[Any, ...],
+        kwargs: Dict[str, Any]
+    ) -> None:
+    """Add positional arguments(args) to keyword argument dictionary(kwargs).
+    Since 'query_items_change_feed' method only allows the following 4 positional arguments in the exact order
+    and types, if the order and types don't match, errors will be raised.
+    If the positional arguments are in the correct orders and types, the arguments will be added to keyword arguments.
+
+    4 positional arguments:
+        - str 'partition_key_range_id': [Deprecated] ChangeFeed requests can be executed against specific partition
+            key ranges. This is used to process the change feed in parallel across multiple consumers.
+        - bool 'is_start_from_beginning': [Deprecated] Get whether change feed should start from
+            beginning (true) or from current (false). By default, it's start from current (false).
+        - str 'continuation': e_tag value to be used as continuation for reading change feed.
+        - int 'max_item_count': Max number of items to be returned in the enumeration operation.
+
+    :param args: Positional arguments. Arguments must be in the following order:
+        1. partition_key_range_id
+        2. is_start_from_beginning
+        3. continuation
+        4. max_item_count
+    :type args: Tuple[Any, ...]
+    :param kwargs: Keyword arguments
+    :type kwargs: dict[str, Any]
+    """
+    if len(args) > 4:
+        raise ValueError(f"Too many arguments. Expected: less than 4, but given: {len(args)}")
```

Comment:
This should be:
`TypeError: foo() takes 4 positional arguments but {len(args)} were given`